doc/config-import.rst

```diff
@@ -957,13 +957,19 @@ This section holds the content type to dlnaprofile mappings.
 
 ::
 
-     <map from="mp4" to="AVC_MP4_BL_CIF30_AAC_MULT5"/>
+    <map from="mp4" to="AVC_MP4_BL_CIF30_AAC_MULT5"/>
 
 * Optional
 
 Specifies a mapping from a certain content type to a dlna profile in the Content Directory. The values of **from** and **to**
-attributes are case sensitive. There are profiles for special codecs which can be access by the pattern ``<content-type>-<video-codec>-<audio-codec>``.
-Video and audio codecs can be seen in the resources properties the web UI.
+attributes are case sensitive.
+In order to access special profiles you can specify a resource attribute with its required value. Mappings with more details 
+are preferred to simple from-to mappings. Resource attributes can be seen in the details page for an item on the web UI.
+
+* Example:
+::
+
+    <map from="mp4" sec:vcodec="h264" sec:acodec="aac" to="AVC_MP4_MP_HD_720p_AAC"/>
```

Comment:
How about `audio_codec` or `audioCodec`, seems a bit "low level" for config

Reply:
This is the spelling of the resource attribute. Each attribute can be referenced here.

Reply:
Yeah but I don't think that is a very friendly thing to have user config

Reply:
How we will we handle the resolution case? Or say SD Vs HD profiles? We would need GT/LT, but resolution is a string.

Simple thing would be to split it to height and width, but no Fields exist for that.

I am thinking something less directly tied to the tags is preferabe

Reply:
Good point. So let's see this as an intermediate step and I think about some transformations for attributes and values to deal with the usability requirement.